native/lambda_game_engine/src/lib.rs

```diff
@@ -30,7 +30,8 @@ fn add_player(game: GameState, character_name: String) -> (GameState, Option<u64
     match game.config.find_character(character_name) {
         None => (game, None),
         Some(character_config) => {
-            let player = Player::new(player_id, character_config);
+            let initial_position = game.config.game.initial_positions[game.players.len()].clone();
```

Comment:
Two problems here
1. This will panic and crash the game if for some reason there are more players than whatever this config says, a crash should never happen 
2. This makes the initial positions static, no randomness on how they are assigned beyond luck on the order you join the game


Elaborating on point 1, I would rather see a `initial_positions` to be a list of points in the map with a radius, this would allow us to spawn in the same point, but slightly different, when repeating an initial position